descriptors/ModuleDescriptor-template.json

```diff
@@ -36,7 +36,7 @@
     },
     {
       "id": "erm",
-      "version": "7.0"
+      "version": "6.1"
```

Comment:
Hi @obozhko-folio version of interface was decreased, could you check is this correct.

Reply:
this is temporary for testing purpose